examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/verifyAnchorHash/verifyAnchorHashCommand.handler.ts

```diff
@@ -23,6 +24,8 @@ export class VerifyAnchorHashCommandHandler
         command.inputForProofVerification,
       );
 
+    this.agent.verifyAnchorHashSignature(command.signature);
```

Comment:
What does it mean to verify anchor hash signature here when the signature is initially (on anchor hash creation) provided with the state, meaning that it has no connection to the anchor hash itself?

Reply:
Are we supposed to verify that the signature stored with the anchor hash actually maps to the pk of the bpi subject that produced the state that was hashed?

Reply:
That was my first though. We could just use the nodejs crypto sign and verify to ensure private key used to sign maps to public key like you said, but we will need the data used to create the signature available on verify side. Since I didn't see the issue mention generation and verification of signature I added those tasks to it. I plan on putting that into a separate PR so it is easier to digest. Thoughts? @ognjenkurtic

Reply:
Agree. Let's do it in a separate PR.